common/djangoapps/student/views.py

```diff
@@ -1735,6 +1736,12 @@ def auto_auth(request):
     reg.activate()
     reg.save()
 
+    # ensure parental consent threshold is met
```

Comment:
Thanks for doing this centrally. It freaked me out seeing this in the core code, but I hadn't realized that auto_auth isn't implemented in our test code.